### DISCUSSION TITLE
Pod is terminated even with lingering enabled

### DISCUSSION THREAD

**Discussion by PinoOG** [containers/podman#21946] [Q&A]
Hello, I have this huge problem, basically I have enabled lingering.
When someone is attached to a pod and while attached logout from SSH, the pod is killed.
**This only happens if someone is attached while logging out!**

Is there a way to solve this problem? I am running a Java Software inside the container with an amazon coretto image.

**Comment by mheon** [comment]
Can you try attaching with `--sig-proxy=false` - I imagine the signal used to kill the attach client is being forwarded to the container

**Comment by PinoOG** [comment]
> --sig-proxy=false

Thanks for helping, tried right now and unluckly din't worked

**Comment by giuseppe** [comment]
that is probably receiving SIGHUP, you either need to detach using the podman `--detach-keys`, by default `c-p c-q`.

You can try wrapping your entrypoint with `nohup`, but if the container process overrides the signal masks then it has no effect

**Comment by PinoOG** [comment]
> that is probably receiving SIGHUP, you either need to detach using the podman `--detach-keys`, by default `c-p c-q`.
> 
> You can try wrapping your entrypoint with `nohup`, but if the container process overrides the signal masks then it has no effect

Thanks for replying, basically I always remember to detach, seems funny but I am trying to solve this issue because of my collaborators who frequently forget to detach before logging out from ssh
